app/controllers/users_controller.rb

```diff
@@ -1,9 +1,14 @@
-class UsersController < ApplicationController
-  def show
-    @user = User.find(params[:id])
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    user_id = params[:user_id]
+    @posts = Post.where(user_id:)
+    @comments = Comment.all
   end
 
-  def index
-    @users = User.all
+  def show
+    @post = Post.find(params[:id])
+    @user = User.find(params[:user_id])
+    @comments = Comment.where(post_id: params[:id])
   end
 end
```

Comment:
- [ ] While you have made a great effort in this milestone, you have put the `PostsController` class in the `users_controller.rb` file which might cause some issues as Rails rely on these naming conventions to associate the controller to the correct view file.